src/testdir/test_filetype.vim

```diff
@@ -551,6 +551,7 @@ def s:GetFilenameChecks(): dict<list<string>>
     pyrex: ['file.pyx', 'file.pxd'],
     python: ['file.py', 'file.pyw', '.pythonstartup', '.pythonrc', 'file.ptl', 'file.pyi', 'SConstruct'],
     ql: ['file.ql', 'file.qll'],
+    qml: ['file.qml'],
```

Comment:
This will need a filetype detection entry in https://github.com/vim/vim/blob/master/runtime/filetype.vim

If *.qbs files are also going to be supported could you please add an entry and tests for that as well?